### PR TITLE
New topic in the docs "Permissions for queries"

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

### CODE REVIEW

File: docs/ru/operations/settings/query_complexity.md [hunk 1]
```diff
@@ -17,18 +17,6 @@
 
 `any (только для group_by_overflow_mode)` - продолжить агрегацию по ключам, которые успели войти в набор, но не добавлять новые ключи в набор.
 
-<a name="query_complexity_readonly"></a>
-
-## readonly
```
Comment (blinkov) [inline docs/ru/operations/settings/query_complexity.md]: I think it's better to keep this while there's no translation of new text.

File: docs/en/operations/settings/permissions_for_queries.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+When using the `GET` method in the [HTTP interface](../../interfaces/http_interface.md#http_interface), `readonly = 1` is set automatically. To modify data use the `POST` method.
+
+<a name="settings_allow_ddl"></a>
+
+## allow_ddl
+
+Allows/denies [DDL](https://en.wikipedia.org/wiki/Data_definition_language) queries.
+
+See the beginning of the page for the division of queries into groups.
```
Comment (blinkov) [inline docs/en/operations/settings/permissions_for_queries.md]: It'll sound weird in single-page mode, better to write "See above for...".

File: docs/toc_en.yml [hunk 3]
```diff
@@ -157,10 +157,10 @@ nav:
     - 'Server settings': 'operations/server_settings/settings.md'
   - 'Settings':
     - 'Introduction': 'operations/settings/index.md'
+    - 'Permissions for queries': 'operations/settings/permissions_for_queries.md'
```
Comment (blinkov) [inline docs/toc_en.yml]: For new articles you need to temporary create symlinks to English version in ru/zh/fa folders and add this line to all `toc_***.yml` files.

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/en/operations/settings/permissions_for_queries.md (hunk 2 above)
Comment (BayoNet) [reply docs/en/operations/settings/permissions_for_queries.md]: Indeed :-). I'll change it.

File: docs/toc_en.yml (hunk 3 above)
Comment (BayoNet) [reply docs/toc_en.yml]: Ок.

**Pull request merged**